packages/dashboard/app/components/__tests__/TaskDetailModal.test-helpers.ts

```
@@ -164,6 +166,16 @@ vi.mock("lucide-react", () => ({
   Image: (props: any) => React.createElement("svg", { "data-testid": "image-icon", ...props }),
   FileText: (props: any) => React.createElement("svg", { "data-testid": "file-text-icon", ...props }),
   FileType: (props: any) => React.createElement("svg", { "data-testid": "file-type-icon", ...props }),
+  /*
+  FNXC:DashboardTests 2026-07-19-01:10:
+  FN-8288/FN-8291 NativeStructurePreview (via StandardChatSurface under TaskDetail)
+  imports Map/Lightbulb/BarChart3/Target for structure cards. Shared TaskDetailModal
+  lucide mock must export them or five modal suites fail at import.
+  */
+  Map: (props: any) => React.createElement("svg", { "data-testid": "map-icon", ...props }),
```

Biome (2.5.3)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)